docs/Documentation/Compatibility.md

```diff
@@ -746,14 +746,15 @@ argument to specify how many of these mobs need to be killed. It's also possible
 You can also add an optional `neutralDeathRadiusAllPlayers` argument to complete the objective for each nearby player
 within the defined radius when the mob is killed by any non-player source.
 You can add a `notify` keyword if you want to send a notification to players whenever the objective progresses.
+You can also add an optional `marked` argument to only count kills marked with the `mspawn` event.
 
 This objective has three properties: `amount`, `left` and `total`. `amount` is the amount of mythic mobs already killed,
 `left` is the amount of mythic mobs still needed to kill and `total` is the amount of mythic mobs initially required.
 
 !!! example
     ```YAML
     mmobkill SkeletalKnight amount:2 events:reward
-    mmobkill SnekBoss amount:2 minlevel:4 maxlevel:6 events:reward
+    mmobkill SnekBoss amount:2 minlevel:4 maxlevel:6 events:reward marked:%player%
```

Comment:
The objective does not support variables for the `marked` parameter.
You can either implement this (which is a bit tricky currently, there is no VariableString like there is VariableNumber) or just fix the example so it doesn't imply that there is variable support.
```suggestion
    mmobkill SnekBoss amount:2 minlevel:4 maxlevel:6 events:reward marked:DungeonBoss3
```